src/components/AddAdvert/AddAdvertForm/AddAdvertForm.jsx

```diff
@@ -0,0 +1,226 @@
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { Section } from './AddAdvertForm.styled';
+import { FormTitle, Button } from '../../common';
+
+
+const initialValues = {
+    title: '',
+    category: '',
+    subcategory: '',
+    time: '',
+    format: '',
+    price: '',
+    language: '',
+    text: '',
+};
+
+
+const AddAdvertForm = () => {
+    const handleSubmit = (values, { resetForm }) => {
+        console.log(values);
+        resetForm();
```

Comment:
It looks like this function is sill in the implementation progress

Reply:
Yes, it doesn`t work now, but submit logic will be here in the future.